caffe2/operators/conv_pool_op_base.h

```diff
@@ -613,6 +606,13 @@ class ConvPoolOpBase : public Operator<Context> {
   }
 
  private:
+ inline void AllocateAndCopy = [&](
+        const vector<int>& vec, Tensor<Context>& tensor) {
```

Comment:
Yes indeed thats just a copy pasta typo :D (trying to figure out how things work on github :D)